widgetssdk/src/main/java/com/glia/widgets/chat/domain/CustomCardShouldShowUseCase.kt

```diff
@@ -0,0 +1,31 @@
+package com.glia.widgets.chat.domain
+
+import com.glia.androidsdk.chat.ChatMessage
+import com.glia.androidsdk.chat.SingleChoiceAttachment
+import com.glia.widgets.chat.adapter.CustomCardAdapter
+
+/**
+ * @hide
+ */
+class CustomCardShouldShowUseCase(private val adapter: CustomCardAdapter?) {
```

Comment:
in Kotlin it's enough to make the class internal, instead of using `@hide`

Reply:
Fixed ef470376e3f759d92b1eba9be9cfdf9fba107080